### ISSUE TITLE
ENV variable descriptions

### ISSUE THREAD

**Issue by ranman034** [kt474/fantasy-football-wrapped#7]
Can you add descriptions on what the env variables represent. So if we want to build locally we know how to populate them. URL paths, tokens, names etc

**Comment by kt474** [comment]
Hi @ranman034! Other than the AI generated weekly reports and viewing trades, the entire site should work without needing any of the environment variables.

- `VITE_USERNAME_URL` and `VITE_USERNAME_URL` are just used to log league ids when they are entered. 
- `VITE_WEEKLY_REPORT` and `VITE_LEAGUE_RECAP` (`https://ffwrapped-backend.vercel.app/api/generateReport`) call my backend which calls the Open AI API to generate a report.  `VITE_PLAYERS_URL` (`https://ffwrapped-backend.vercel.app/api/getPlayer`) retrieves player names from my database.  Even if these values were set locally they would be blocked by CORS. Hopefully I can eventually make the backend public too so users can use their own Open AI token and db.

**Comment by ranman034** [comment]
Ah I see. Yea I wanted to mess around with the AI report because it returns some incorrect info on a regular basis

**Comment by kt474** [comment]
> Ah I see. Yea I wanted to mess around with the AI report because it returns some incorrect info on a regular basis

Thanks for letting me know. I'll work on improving the prompt. 

For the playoff matchups here's the prompt and input I'm using. I'm pretty new to this stuff so I'm open to suggestions - there are probably more inputs I should give for each matchup. 

```
You are a fantasy football analyst with a sharp, funny trash-talking tone. Give a concise, witty, insightful 
weekly summary with no more than 215 words. Teams with the same matchupNumber are opponents. 
```

Input: 

```
leagueMetadata: {
    playoffRound:  "Quarterfinal round"
},
matchups: [{
    inLosersBracket: false,
    inWinnersBracket: true, 
    matchupNumber: 2, 
    name: "...",
    playerNames: ["Justin Herbert", "James Conner", "David Montgomery", "Tyreek Hill", "Drake London", "Brock Bowers",…],
    playerPoints: [13.8, 28.3, 5.5, 4.6, 12.8, 5, 15.7, 8.8, 6], 
   etc ... 
}]
```

**Comment by ranman034** [comment]
What I noticed was it talked about the player points but would say the incorrect team won the matchup

**Comment by kt474** [comment]
> What I noticed was it talked about the player points but would say the incorrect team won the matchup

Yeah I've noticed that as well, I'll work on updating the prompt/inputs and let you know when it's improved. Thanks again for the feedback!

**Comment by kt474** [comment]
@ranman034 I've updated the prompt and inputs so the AI shouldn't mix up the winning/losing team any more. Please let me know if you run into any other issues.

**Comment by ranman034** [comment]
Awesome. I'll take a look after this week finishes and get back to you
